bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
+
+    @classmethod
+    def setup_dependencies(cls):
```

Comment:
I believe the `Bear.__init__` logic is:
```
        self.section = section
        ...
        self.setup_dependencies()
        cp = type(self).check_prerequisites()
```
(I may not be correct, especially for `GlobalBear`s)

That means the `setup_dependencies` is called for each instance.
As a `GlobalBear`, this bear is typically only run once per `coala` invocation, so `setup_dependencies` will typically only be called once.

It seems odd that a class method is used to do the `check_prerequisites`, as that means there can not be instance- (configuration-) specific prerequisites.  And the class method `check_prerequisites` is being invoked for every instance.  It should be in `__new__` , or in the meta-class, as it can't be different each instantiation..?

`self.section` is being set before `setup_dependencies` is invoked , so I might be able to get `check_imperative` from there.  (I did think of this, but was being lazy so I didnt investigate beyond wondering why we have `setup_dependencies` and `check_prerequisites` (and `prerequisite_check_command`: https://github.com/coala/coala/issues/2073), and especially why have them in that order, and wtf for the class method.)